### PR TITLE
Updated summary api to include order and name

### PR DESCRIPTION
**Pull request by goyal1092**
## Description
Updated summary api with orders of subcategory, category and profile indicator

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-277

## How to test it locally
Request summary api endpoint and it should show order and name as new keys for every subcategory, category and profile indicator

## Checklist

- [x]  🚀 is the code ready to be merged and go live?
- [x]  🛠 does it work (build) locally

### Pull Request

- [x]  📰 good title
- [x]  📝good description
- [x]  🔖 issue linked
- [x]  📖 changelog filled out

### Commits

- [x]  commits are clean
- [x]  commit messages are clean

### Code Quality

- [x]  🚧 no commented out code
- [x]  🖨 no unnecessary logging
- [x]  🎱 no magic numbers
- [x]  black was run locally (as part of the pre-commit hook)

### Testing

- [x]  ✅ added (appropriate) unit tests
- [x]  💢 edge cases in tests were considered
- [x]  ✅ ran tests locally & are passing

### CODE REVIEW

File: wazimap_ng/profile/serializers/utils.py [hunk 1]
```diff
@@ -43,7 +43,8 @@ def get_indicator_data(profile, indicators, geographies, version):
                 content_type=F("indicator__profileindicator__content_type"),
                 chart_type=F("indicator__profileindicator__chart_type"),
                 choropleth_range=F("indicator__profileindicator__choropleth_range"),
-                enable_linear_scrubber=F("indicator__profileindicator__enable_linear_scrubber")
+                enable_linear_scrubber=F("indicator__profileindicator__enable_linear_scrubber"),
+                pi_order=F("indicator__profileindicator__order")
```
Comment (michaelglenister) [inline wazimap_ng/profile/serializers/utils.py:47]: pi_order should rather be profile_indicator__order, if just to be consistent with other variable naming

**Review by michaelglenister — COMMENTED** [review]
I think the test covers the code change. Just a variable name change and it's good to go

File: wazimap_ng/profile/serializers/utils.py (hunk 1 above)
Comment (michaelglenister) [inline wazimap_ng/profile/serializers/utils.py:47]: To be consistent I think this should be
```suggestion
                profile_indicator_order=F("indicator__profileindicator__order")
```

**Review by michaelglenister — COMMENTED** [review]
One change suggestion, otherwise I'm happy with the changes
